commit 3ce0f54fe73e45c89f20bf6210aea05ebeaa12d8
Author: davidw5 <37127909+davidw5@users.noreply.github.com>
Date:   Thu Mar 8 09:23:05 2018 +1100

    Update dts Makefile to include overlays directory
    
    Add "subdir-y += overlays" to arch/arm64/boot/dts/Makefile to enable 'make ARCH=arm64 dtbs' to generate overlay .dtbo files

diff --git a/arch/arm64/boot/dts/Makefile b/arch/arm64/boot/dts/Makefile
index 4aa50b9b26bc..747bbada505a 100644
--- a/arch/arm64/boot/dts/Makefile
+++ b/arch/arm64/boot/dts/Makefile
@@ -24,3 +24,5 @@ subdir-y += socionext
 subdir-y += sprd
 subdir-y += xilinx
 subdir-y += zte
+
+subdir-y += overlays
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1167_3ce0f54fe73e45c89f20bf6210aea05ebeaa12d8.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1167_3ce0f54fe73e45c89f20bf6210aea05ebeaa12d8.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
